lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
+-- @copyright 2017 nahsi
+-- @module awful.hotkeys_popup.keys.tmux
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local tmux_rule = { name = "tmux" }
```

Comment:
Oh well, then I'll do some research on how to implement what you suggested (I don't know lua or any other programming language) + some sort of default option if there is no function call.